Memento/AddEmployeeToManageList.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Command.Interfaces;
+using Memento;
+
+namespace Command
+{
+    public class AddEmployeeToManageList : ICommand
+    {
+        private readonly int managerId;
+        private readonly Employee employee;
```

Comment:
Why is the managerId not part of Employee class?

Reply:
I thought about it, but I wanted to keep the Employee class cleaner so that it would also fit other cases